Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+for a small propulsion system, and an understanding of the design complexites. These can help avoid roadblocks in future designs. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy and it is very safe to show to people when the tank is unpressurized. This project
+will also be the first foot in the door for CubeSat propulsion. There is also a continuation of this project ready to be initated upon its completion, the
+reaction control system.
+
+There are many levels of
+analysis that can be used to design one, but the best way is experimentation (trial \& error)\cite{RPE}. This is especially true for cold gas propulsion of this
+size. The combination of the size of the nozzle and the high reynolds number of the gas does not allow and analytical solution to the optimal dimensions
+of the nozzle. An analytical solution usually concerns making a conical nozzle. However the thrust loss on these nozzles is maxmized. This is due to the
+direction of the outgoing flow. The particles travel away from the nozzle at an angle and this increases the horizontal velocity component, and decreases the
+vertical velocity (the direction pointing out of the nozzle), which decreases the thrust of the system\cite{SPAD}. Since cold gas propulsion is inherently inefficient, due
+to the fact that their is no chemcial reaction or heat applied to the system, maxmizing the thrust is imperitive.
+
+Last, but certainly not least, is the nozzle determination experience. Because of the flow conditions and dimensions, an analytical solution will not be sufficient. To accurately determine
+the nozzle size experiementation must happen. This data will be immensely helpful for designing the RCS nozzles and any other CGT nozzles with similar
+size magnitudes.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include 4 to 5 weeks of design, 3 to 5 weeks of fabrication, and 4 to 5 weeks of data analysis. The project will start off with a redesign discussion of the test stand, optmizing it for new conditions if necessary.
+The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+nozzle dimensions to create the polynomial curve approximation that will be an integral part of the rest of the research on the topic. Once that is complete, the
+testing and experimentation phase begins. At the end of the data analysis the final report will be the main focus. The aim is to complete it by the beginning of winter break
```

Comment:
This can be reworded

>At the end of the data analysis process, the team will generate a final report including test procedures, design choices, and test data.